### ISSUE TITLE
CI failing on snapshot versions not finding transient

### ISSUE THREAD

**Issue by ericdallo** [ericdallo/jet.el#1]
https://github.com/ericdallo/jet.el/actions/runs/4009420016/jobs/6884750236

Not sure if it's a Eask issue, maybe @jcs090218 knows?

**Comment by jcs090218** [comment]
`transient` is part of the Emacs core now, so you can simply remove

```diff
- ;; Package-Requires: ((emacs "27.1") (transient "0.3.7"))
+ ;; Package-Requires: ((emacs "27.1"))
```

But I am not quite sure if this only happens in snapshot, i will investigate it.

**Comment by ericdallo** [comment]
Yeah, that's the problem, for older emacs like 27.1 we still need the transient package, but not for the snapshot one

**Comment by jcs090218** [comment]
I have published a new version for this `0.7.9`, can you try to re-run the workflow?

**Comment by ericdallo** [comment]
It works! thank you @jcs090218 !
